epoxy-paging/src/androidTest/java/com/airbnb/epoxy/paging/PagedListModelCacheTest.kt

```diff
@@ -236,6 +236,13 @@ class PagedListModelCacheTest {
     }
   }
 
+  @Test
+  fun clear() {
+    pagedListModelCache.clearModels()
```

Comment:
afaik this doesn't test that models are cleared, just that models are rebuilt. I think a good way to test the behavior is to check that all models are rebuilt - maybe by adding a counter in the `modelBuilder` callback similar to how `rebuildCounter` is used

Reply:
Fixed the test. The callback call in the previous message was hiding the fact that the cache didn't even have models.